### PR TITLE
Gift Entry: Fabricate Donor Name from Lookup

### PR DESCRIPTION
**Pull request by bdvorachek**
# Critical Changes

# Changes
- When the First/LastName or Account Name fields are missing from the Gift Entry form, use the values in the Donor Lookup field to populate cardholder name if Payment Widget is present.
# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by bdvorachek** [comment]
**lurch: attach W-041099

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HvVgQAK" target="_blank">W-041099</a>

File: src/lwc/geFormRenderer/geFormRenderer.js [hunk 1]
```diff
@@ -1585,35 +1584,46 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
         }
     }
 
+
     handleNameOnCardFieldChange() {
+        this.setCardHolderName();
```
Comment (voduyemi) [inline src/lwc/geFormRenderer/geFormRenderer.js:1589]: @bdvorachek We need to be able to read the card holder name `onChange` of the Donor Type field of the form and when the `firstName` and `lastName` fields are not on the form, it seems the dataImport record only gets constructed onLookUpRecordSelect. Depending on the order in which you populate data in the form, the names do not get fabricated. I think we could queue up the selected `recordId` for retrieval using the `queueSelectedRecordForRetrieval` function

**Comment by bdvorachek** [comment]
Closing this one - see #5460.

**Pull request closed without merging**